process/outportBlockConverter.go

```diff
@@ -384,6 +386,44 @@ func handleHeaderGasConsumption(consumption *outport.HeaderGasConsumption, shard
 	}
 }
 
+func handleStateChanges(stateChanges map[string]*stateChange.StateChanges, shardOutportBlock *hyperOutportBlocks.ShardOutportBlockV2) {
+	if stateChanges == nil {
+		return
+	}
+
+	shardStateChanges := make(map[string]*hyperOutportBlocks.StateChanges, len(stateChanges))
+	for key, scs := range stateChanges {
+		ss := make([]*hyperOutportBlocks.StateChange, len(scs.StateChanges))
+
+		for i, sc := range scs.StateChanges {
+			dtc := make([]*hyperOutportBlocks.DataTrieChange, len(sc.DataTrieChanges))
+
+			for j, dataTrieChange := range sc.DataTrieChanges {
+				dtc[j] = &hyperOutportBlocks.DataTrieChange{
+					Type: dataTrieChange.Type,
+					Key:  dataTrieChange.Key,
+					Val:  dataTrieChange.Val,
+				}
+			}
+
+			ss[i] = &hyperOutportBlocks.StateChange{
+				Type:            sc.Type,
+				Index:           sc.Index,
+				TxHash:          sc.TxHash,
+				MainTrieKey:     sc.MainTrieKey,
+				MainTrieVal:     sc.MainTrieVal,
+				Operation:       sc.Operation,
+				DataTrieChanges: dtc,
+			}
+		}
+
+		s := &hyperOutportBlocks.StateChanges{StateChanges: ss}
+		shardStateChanges[key] = s
```

Comment:
please avoid single letter variable here, maybe set it directly

Reply:
changed.